tools/copyblocks/README.md

```diff
@@ -0,0 +1,9 @@
+# Copy blocks
+
+This program can copy Mimir blocks between two GCS buckets. It is GCS specific and uses special calls to copy
+files between buckets directly, without download to a local system first.
+
+copyblocks can run in one-time mode, or as a service and do regular copies.
+
+It can be configured with minimum block time range to avoid copying of blocks that are too small.
+It can also be configured with list of users to copy, or avoid copying blocks for.
```

Comment:
```suggestion
You can also list users to copy, or avoid copying blocks for.
```
What does, "or avoid copying blocks for" mean? Cannot parse this meaning as-is.

Reply:
You can configure an allow list and a block list. By default all tenants are copied.